### PR TITLE
Gives the bar privacy shutters so that the bartender can actually close down their bar

### PR DESCRIPTION
**Pull request by athena413**
# General Documentation

### Intent of your Pull Request

This gives the bar actual privacy shutters over the tables in the bar, which lets the bartender close their bar down without needing to worry about people hopping the tables and greytiding their shit while the barman is gone.

Includes:
IceBox
Meta
Delta
Omega
Eclipsestation (all variants)
Box (all variants)
Space Bartender ghost role
Lavaland beach biodome bartender ghost role

### Why is this change good for the game?

bartenders can lower their shutters to get greytided less

mmmm shutter crush monke funny

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
there's shutters now

### What should players be aware of when it comes to the changes your PR is implementing?
you can open and close them like a door

### What general grouping does this PR fall under? 
mapping

### Are there any aspects of the PR that you would like us not to mention on the Wiki?
no

# Changelog

:cl:  
rscadd: shutters, and a lot of them
/:cl:

### CODE REVIEW

**Comment by UselessTheremin** [comment]
sure, cool

**Review by patpol4 — CHANGES_REQUESTED** [review]
You used the poddoors instead of shutters in most of these, here is a comparison:
vivaldi_zhOsFkCQKI
on the left poddoors, heavy, solid, high security, we dont want these for a bartender
on the right shutters, they look lighter and more like something a bar would have instead of a vault.
Please swap them to be shutters.

**Review by patpol4 — APPROVED** [review]
Changed the names and access on the buttons because they werent capitalized and had chef access.
Some of this looks bad but we'll see how it goes.
